Demo/Navigator.swift

```diff
@@ -0,0 +1,10 @@
+import Foundation
+import Turbo
+
+/// A bridge "back" to Turbo world from native.
+/// See `NumbersViewController` for an example of navigating from native to web.
+protocol Navigator: AnyObject {
+    func route(_: URL)
+}
```

Comment:
I'd love some feedback on this approach. Ideally we can go from native screens back to Turbo Navigator without a protocol.

Reply:
In order to prevent `TurboNavigator` from being deeply coupled, using a protocol is a good idea. That concern is at the app-level, not the framework level, so I don't see a problem with keeping it in the Demo.

Reply:
Cool, thanks for the feedback!